cron/internal/worker/main.go

```diff
@@ -161,7 +196,16 @@ func processRequest(ctx context.Context,
 		if err != nil {
 			return fmt.Errorf("error during policy.GetEnabled: %w", err)
 		}
-		for _, check := range blacklistedChecks {
+
+		disabledChecks := blacklistedChecks
+		if _, err := gitlabrepo.MakeGitlabRepo(*repoReq.Url); err != nil {
```

Comment:
can we save the results from above where we set the gitlab repos? or make this logic be in the same place?